pulp_smash/tests/pulp2/rpm/api_v2/test_errata.py

```diff
@@ -0,0 +1,87 @@
+"""Test whether applying an ``Errata`` will update an RPM package version."""
```

Comment:
Nitpick: "Errata" isn't a proper noun, nor is it code. It's fine to reference "errata."